nkundajanv02/my-brand
# #VVi09Bfc Implementing home page

#### What does this PR do?
This PR implements the Home page. 

#### Description of Task to be completed?
-add My name
-Add a nav bar with the following links:
    *Home
   *About
   *Blog
   *Skills
   *Works
   *Contacts
   *Login
   *Download CV
-Add introductory text
-Add an image summarizing my skills 
-Add my three working areas and corresponding images
-add a contact me button
     
#### How should this be manually tested?
-Clone the [repo](https://github.com/nkundajanv02/my-brand)
-Checkout branch `feat-implement-home-page-VVi09Bfc`
-Open the index.html in your browser to check the new homepage

#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[#VVi09Bfc](https://trello.com/c/VVi09Bfc/19-home)

#### Screenshots
N/A
